### DISCUSSION TITLE
Hide titles and only show task icons

### DISCUSSION THREAD

**Discussion by neilroy1998** [lawand-dot-io/taskbar#10] [Feature requests]
Just showing the icon would be amazing with an indicator. This helps to minimize space when too many windows are open

**Comment by lawand-dot-io** [comment]
I'll think about this, it will make it difficult to know which is window is which. But this is worth considering. For the moment, one thing you can do is to make the size of the tasks smaller. There's an option in preferences and so you can make the window title really small and get a behaviour that's close to what you're asking :)

**Reply by neilroy1998** [reply]
I think this will also help in getting the taskbar to left or right! And as for the Window title, just show it when one hovers over the app (below the thumbnails)...! _Basically like Windows 10/11_

**Comment by lawand-dot-io** [comment]
This feature has been implemented in Taskbar version 1.2.1 which is now released. Closing the ticket.
@neilroy1998
